src/bit-systems/media-loading.ts

```diff
@@ -176,13 +178,14 @@ type MediaInfo = {
   thumbnail: string;
 };
 
+const loadingObjs = new Map<EntityID, EntityID>();
+
 function* loadMedia(world: HubsWorld, eid: EntityID) {
-  let loadingObjEid = 0;
   const addLoadingObjectTimeout = crTimeout(() => {
-    loadingObjEid = renderAsEntity(world, LoadingObject());
+    const loadingObjEid = renderAsEntity(world, LoadingObject());
+    loadingObjs.set(eid, loadingObjEid);
     add(world, loadingObjEid, eid);
   }, 400);
-  yield withRollback(Promise.resolve(), () => loadingObjEid && removeEntity(world, loadingObjEid));
```

Comment:
@keianhzo @johnshaughnessy Can you explain what this line was for? I don't get and am not sure if removing this line is safe or not.